src/parser/core/modules/Interrupts.tsx

```diff
@@ -0,0 +1,143 @@
+import {MessageDescriptor} from '@lingui/core'
+import {t} from '@lingui/macro'
+import {Trans} from '@lingui/react'
+import React from 'react'
+import {Button, Table} from 'semantic-ui-react'
+
+import {ActionLink} from 'components/ui/DbLink'
+import ACTIONS from 'data/ACTIONS'
+import {CastEvent} from 'fflogs'
+import Module, {dependency} from 'parser/core/Module'
+import GlobalCooldown from 'parser/core/modules/GlobalCooldown'
+import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+import Timeline from 'parser/core/modules/Timeline'
+import {Data} from './Data'
+
+interface SeverityTiers {
+	[key: number]: number
+}
+
+// used for timeline viewing by giving you a nice 30s window
+const TIMELINE_UPPER_MOD: number = 30000
+
+export abstract class Interrupts extends Module {
+	static handle: string = 'interrupts'
+	static title: MessageDescriptor = t('core.interrupts.title')`Interrupted Casts`
+
+	@dependency private timeline!: Timeline
+	@dependency private globalCooldown!: GlobalCooldown
+	@dependency private suggestions!: Suggestions
+	@dependency private data!: Data
+
+	private currentCast?: CastEvent | undefined
+	private droppedCasts?: CastEvent[] = []
```

Comment:
Since this is being assigned, you can straight up `droppedCasts` without the optional indicator.